src/zui/ZUIPersonHoverCard.tsx

```diff
@@ -92,6 +94,17 @@ const ZUIPersonHoverCard: React.FunctionComponent<{
                     id={person?.id}
                     link
                     name={`${person?.first_name} ${person?.last_name}`}
+                    subtitle={
```

Comment:
When we looked at this today, we realized that the subtitle gets a hover effect similar to the title, which looks ood because of the coloring. Can you look into that @ziggabyte?